20231212-modular-models.md

```diff
@@ -227,6 +227,81 @@ confluence.model.fga @atlassian/confluence
 
 The OpenFGA server does not need to be changed, beyond supporting the new JSON format and allowing schema version `1.2`. SDKs do not need to be changed beyond being refreshed to allow for the new Authorization Model type with the additional fields.
 
+### Type Extension
+
+With one of the main drivers behind modular models being to help reduce friction between teams working on a singular model it was decided that type extension should be included to allow extending types without the need for the owning team to take ownership of these relations. Moving forward, it should be feasible to allow restricting the usage of these relations to within the module they are extended within.
```

Comment:
```suggestion
With one of the main drivers behind modular models being to help reduce friction between teams working on a singular model it was decided that type extension should be included to allow extending types without the need for the owning team to take ownership of these relations. In the future, it would be possible to allow restricting the usage of these relations to within the module they are extended within.
```